articles/azure/azs-ref-overview.md

```diff
@@ -158,23 +170,23 @@ You can access UKCloud for Microsoft Azure from the following location:
 
 For more information about Azure Stack Hub, see the following Microsoft resources:
 
-- [*Get Started with Azure*](https://azure.microsoft.com/en-gb/get-started/)
-
-- [Configure the Azure Stack Hub user's PowerShell environment](azs-how-configure-powershell-users.md)
+- [Get Started with Azure](https://azure.microsoft.com/en-gb/get-started/)
```

Comment:
Above it says information about Azure Stack Hub, but the link is to Azure doc...

Reply:
Updated link in commit 7826a84.  Thank you for the review :)